docs/docs/reference/impersonation.md

```diff
@@ -5,7 +5,7 @@ description: >-
   user to impersonate another user or group.
 ---
 
-# User impersonation
+# User impersonation & Service Accounts
 
```

Comment:
tiny suggestion: `User Impersonation & Service Accounts` to be consistent.

Reply:
👍